packages/node/src/parseForConfig.js

```diff
@@ -86,12 +101,12 @@ function makeImportHook ({
     const extension = path.extname(filename)
     const packageData = packageDataForModule({ id: specifier, file: filename }, rootPackageName)
 
-    if (extension === '.node') {
-      return makeNativeModuleRecord(specifier, filename, packageData)
-    }
     if (['.js', '.cjs'].includes(extension)) {
       return makeJsModuleRecord(specifier, filename, packageData)
     }
+    if (extension === '.node') {
+      return makeNativeModuleRecord(specifier, filename, packageData)
```

Comment:
What's the purpose of shifting this below?

Reply:
its less common, so we'll hit the common case first. likely a tiny but unmeasured perf boost